src/FixMath.h

```diff
@@ -1105,6 +1115,11 @@ class SFix
       @return The number of bits used to encode the fractional part.
   */
   static constexpr int8_t getNF() {return NF;}
+
+    /** The (absolute) range of the integral part of the number
```

Comment:
Does this have a real-world benefit? If yes, I suggest to clarify the documentation to avoid the misconception that this is the actual range. E.g.:

"The (absolute) range of the integral part of the number, as far as it can be determined at compile time. The true range of the number will typically be lower than (at most the same as) this, so it is not recommended to use this in computations."

Reply:
> Does this have a real-world benefit? 

Not really, but this is useful for debugging purposes. Maybe would be useful to put in a `#if FIXMATH_DEBUG` macro?